src/scriptdev2/ScriptMgr.cpp

```diff
@@ -89,56 +87,48 @@ void FreeScriptLibrary()
 MANGOS_DLL_EXPORT
 void InitScriptLibrary()
 {
-    // ScriptDev2 startup
-    outstring_log("");
-    outstring_log(" MMM  MMM    MM");
-    outstring_log("M  MM M  M  M  M");
-    outstring_log("MM    M   M   M");
-    outstring_log(" MMM  M   M  M");
-    outstring_log("   MM M   M MMMM");
-    outstring_log("MM  M M  M ");
-    outstring_log(" MMM  MMM  http://www.scriptdev2.com");
-    outstring_log("");
-
-   // Get configuration file
-   bool configFailure = false;
-   if (!SD2Config.SetSource(_MANGOSD_CONFIG))
-       configFailure = true;
-   else
-       outstring_log("SD2: Using configuration file %s", _MANGOSD_CONFIG);
+   // ScriptDev2 startup
+   outstring_log("");
+   outstring_log(" MMM  MMM    MM");
+   outstring_log("M  MM M  M  M  M");
+   outstring_log("MM    M   M   M");
+   outstring_log(" MMM  M   M  M");
+   outstring_log("   MM M   M MMMM");
+   outstring_log("MM  M M  M ");
+   outstring_log(" MMM  MMM  http://www.scriptdev2.com");
```

Comment:
Now you're at it we probably wanna update this to the cmangos SD2 subforum, (link is in the README)